odoo-shippable/scripts/build-image.sh

```diff
@@ -121,14 +124,22 @@ git_clone_execute "${OH_MY_ZSH_REPO}" "master" "tools/install.sh"
 git_clone_copy "${ZSH_THEME_REPO}" "master" "schminitz.zsh-theme" "~/.oh-my-zsh/themes/odoo-shippable.zsh-theme"
 sed -i 's/robbyrussell/odoo-shippable/g' ~/.zshrc
 
-# Install & configure vim
+# Upgrade & configure vim
+apt-get upgrade vim
```

Comment:
Upgrade of vim is required because the package is it kept back! https://travis-ci.org/Vauxoo/docker-odoo-image/builds/192995602#L4257
@moylop260 What do you think of the use of `apt-get upgrade vim`?